styles/globals.scss

```diff
@@ -6,8 +6,22 @@
 
 html,
```

Comment:
Lots of styling tweaks, sorry

Reply:
I had been mixing up em/rem - rem is the one that is set through the page as compared to em which is set by the parent element. we want rem

Reply:
To confirm, we want rem so that the page can be proportional to whatever the browser determines is the "baseline" spacing?